packages/backend/src/App.ts

```diff
@@ -453,6 +457,9 @@ export default class App {
         expressApp.use(
             (error: Error, req: Request, res: Response, _: NextFunction) => {
                 const errorResponse = errorHandler(error);
+                if (error instanceof UnexpectedServerError) {
+                    Logger.error(error); // Log original error for debug purposes
```

Comment:
Is this intended ? 
Recently, for debugging I've been using `console.error` instead of  `Logger.error` because with console we show the full stacktrace. Or add `console.trace(e)` on top.